index.js

```diff
@@ -2651,27 +2651,29 @@
   }
   S.find = def('find', {}, [$.Function, $.Array(a), $Maybe(a)], find);
 
-  //# pluck :: Accessible a => TypeRep b -> String -> Array a -> Array (Maybe b)
+  //# pluck :: Accessible a => String -> Array a -> Array b
   //.
-  //. Takes a [type representative](#type-representatives), a property name,
-  //. and an array of objects and returns an array of equal length. Each
-  //. element of the output array is Just the value of the specified property
-  //. of the corresponding object if the value is of the specified type
-  //. (according to [`is`](#is)); Nothing otherwise.
-  //.
-  //. See also [`get`](#get).
+  //. Takes a property name and an array of objects and returns an array in
+  //. which each element is the value of the specified property of the
+  //. corresponding object. If for some reason the object lacks the specified
+  //. property, a type error is thrown.
   //.
   //. ```javascript
-  //. > S.pluck(Number, 'x', [{x: 1}, {x: 2}, {x: '3'}, {x: null}, {}])
-  //. [Just(1), Just(2), Nothing, Nothing, Nothing]
+  //. > S.pluck('x', [{x: 1}, {x: 2}, {x: 3}])
+  //. [1, 2, 3]
   //. ```
-  function pluck(typeRep, key, xs) {
-    return xs.map(function(x) { return get(typeRep, key, x); });
+  function pluck(key, xs) {
+    return xs.map(function(obj, idx) {
+      if (key in Object(obj)) return obj[key];
+      throw new TypeError('‘pluck’ expected object at index ' + idx + ' to ' +
+                          'have a property named ‘' + key + '’; ' +
```

Comment:
Let's move `' to '` down one line for more pleasing line wrapping (saving one concatenation operation in the process).